lbrynet/lbrynet_daemon/LBRYDaemonCLI.py

```diff
@@ -35,7 +35,7 @@ def main():
     if len(args) > 1:
         if isinstance(args[1], dict):
             params = args[1]
-        elif isinstance(args[1], str, unicode):
+        elif isinstance(args[1], str) or isinstance(args[1], unicode):
```

Comment:
can also use a tuple for the classes: https://docs.python.org/2/library/functions.html#isinstance